pkg/monitoring/config/parse.go

```diff
@@ -105,6 +119,31 @@ func applyDefaults(cfg *Config) {
 	}
 }
 
+func validateConfigWithoutKafka(cfg Config) error {
+	// Required config
+	for envVarName, currentValue := range map[string]string{
+		"FEEDS_URL": cfg.Feeds.URL,
+		"NODES_URL": cfg.Nodes.URL,
+
+		"HTTP_ADDRESS": cfg.HTTP.Address,
+	} {
+		if currentValue == "" {
+			return fmt.Errorf("'%s' env var is required", envVarName)
+		}
+	}
+	// Validate URLs.
+	for envVarName, currentValue := range map[string]string{
+		"FEEDS_URL": cfg.Feeds.URL,
+		"NODES_URL": cfg.Nodes.URL,
+	} {
+		if _, err := url.ParseRequestURI(currentValue); err != nil {
+			return fmt.Errorf("%s='%s' is not a valid URL: %w", envVarName, currentValue, err)
+		}
+	}
+
+	return nil
+}
+
 func validateConfig(cfg Config) error {
 	// Required config
 	for envVarName, currentValue := range map[string]string{
```

Comment:
similarly here, can this be rewritten in terms of the common (without kafka) validation and the extra (kafka-specific) validation?

~
```

func validateConfigWithKafka() {
 err := baseValidation(cfg)
 ...
 // kafka stuff
 }	
 
 ```
 
 where i've renamed validateConfigWithoutKafka to baseValidation